submissions/Yasya23/tiny-js-oop/index.js

```diff
@@ -57,5 +54,5 @@ const woman = new Human("Anna", "female", "Good luck!");
 const inhabitants = [dog, cat, catWoman, man, woman];
 
 inhabitants.forEach(function (inhabitant) {
-  print(inhabitant.showProperties());
+  print(inhabitant.showProperties().join("; "));
 });
```

Comment:
Arrow functions will make code less verbose.
Compare:
```
inhabitants.forEach(function (inhabitant) {
  print(inhabitant.showProperties().join("; "));
});

inhabitants.forEach(inhabitant => print(inhabitant.showProperties().join("; ")));
```

Reply:
Thanks for checking. Yes, it looks much better, fixed it.